### PR TITLE
IoT dashboard

### PR DESCRIPTION
**Pull request by abrarqa62**
This is my Complete IoT Dashboard Automation Video :

https://user-images.githubusercontent.com/106743757/174845831-e76d2f83-b11c-4114-b639-675a9713e9a4.mp4

### CODE REVIEW

File: cypress/integration/IoTDashboard.spec.js [hunk 1]
```diff
@@ -0,0 +1,420 @@
+import { ToNavigateIoT } from "../page_objects/PageObjectIoTdashboard";
+
+describe("Iot Dashboad", () => {
+  before("User move to IoT Dashboard", () => {
+    ToNavigateIoT.IotDashboard().click();
```
Comment (hassanatdevsinc) [inline cypress/integration/IoTDashboard.spec.js:5]: please add assertion here to verify that the main page is loaded and make sure IotDashboard is visible then click on it.

File: cypress/integration/IoTDashboard.spec.js [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+  });
+  context("When user is on Status Card", () => {
+    it("Verify if user can click on Light Icon to OFF light", () => {
+      ToNavigateIoT.LightBulb().click();
```
Comment (hassanatdevsinc) [inline cypress/integration/IoTDashboard.spec.js:9]: please add should be visible on every element before clicking on the element.

File: cypress/integration/IoTDashboard.spec.js [hunk 3]
```diff
@@ -0,0 +1,420 @@
...
+    });
+    it("Verify if user can click on Light Icon to ON light", () => {
+      ToNavigateIoT.LightBulb().click();
```
Comment (hassanatdevsinc) [inline cypress/integration/IoTDashboard.spec.js:12]: please add an assertion to verify that action has been performed I.e element has changed

File: cypress/integration/IoTDashboard.spec.js [hunk 4]
```diff
@@ -0,0 +1,420 @@
...
+    });
+    it("Verify if user can click on Roller Shades Icon to OFF roller shades", () => {
+      ToNavigateIoT.Rollershades().click();
```
Comment (hassanatdevsinc) [inline cypress/integration/IoTDashboard.spec.js:15]: there would assertion to verify 1st shades icon to be before clicking

File: cypress/integration/IoTDashboard.spec.js [hunk 5]
```diff
@@ -0,0 +1,420 @@
...
+    });
+    it("Verify if user can click on Coffer Maker Icon to OFF Coffee Maker", () => {
+      ToNavigateIoT.CoffeeMaker().click();
+    });
+    it("Verify if user can click on Coffee Maker Icon to ON Coffee Maker", () => {
+      ToNavigateIoT.CoffeeMaker().click();
+    });
+  });
+
+  // Temperature Section
```
Comment (hassanatdevsinc) [inline cypress/integration/IoTDashboard.spec.js:34]: no need for a comment here as we have a context here

**Review by hassanatdevsinc — CHANGES_REQUESTED** [review]

File: cypress/integration/IoTDashboard.spec.js (hunk 3 above)
Comment (abrarqa62) [reply cypress/integration/IoTDashboard.spec.js:12]: Done

File: cypress/integration/IoTDashboard.spec.js (hunk 4 above)
Comment (abrarqa62) [reply cypress/integration/IoTDashboard.spec.js:15]: Done

File: cypress/integration/IoTDashboard.spec.js (hunk 5 above)
Comment (abrarqa62) [reply cypress/integration/IoTDashboard.spec.js:34]: Done

File: cypress/integration/IoTDashboard.spec.js (hunk 2 above)
Comment (abrarqa62) [reply cypress/integration/IoTDashboard.spec.js:9]: Done

File: cypress/integration/IoTDashboard.spec.js (hunk 1 above)
Comment (abrarqa62) [reply cypress/integration/IoTDashboard.spec.js:5]: Done

**Pull request merged**